core/src/main/java/io/questdb/ServerMain.java

```diff
@@ -223,11 +223,13 @@ public void close() {
                 // Still useful in case of custom logger
                 bootstrap.getLog().info().$("QuestDB is shutting down...").$();
             }
+            freeOnExit.close();
```

Comment:
With this change all of our `close()` methods for the freed-on-exit resources need to assume concurrent access. Currently, that's not the case and most of these methods assume no concurrency.

Reply:
that is correct, change reverted